docs/data/hubble/analyst-guide/queries-for-horizon-like-data.mdx

```diff
@@ -0,0 +1,922 @@
+from `crypto-stellar.crypto_stellar.history_transactions`
+where true
+  and id = <transaction id>
+  -- transaction_hash is also available
+  --and transaction_hash = <transaction hash>
+  -- highly recommended to provide a date range to reduce amount of data queried
+  and closed_at between <start> and <end>
+```
+
+### Retrieve a Transaction’s Operations
+
+```sql
+select
+  transaction_id
+  , op_id
+  , max_fee
+  , operation_count
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar_dbt.enriched_history_operations`
+where true
+  and transaction_id = <transaction id>
+  -- transaction_hash is also available
+  --and transaction_hash = <transaction hash>
+  -- highly recommended to provide a date range to reduce amount of data queried
+  and closed_at between <start> and <end>
+```
+
+## Trade Aggregations
+
+[horizon/trade_aggregations](/docs/data/horizon/api-reference/aggregations/trade-aggregations)
+
+A trade aggregation represents aggregated statistics on an asset pair (base and counter) for a specific time period. Trade aggregations are useful to developers of trading clients and provide historical trade data.
+
+Hubble does not have as granular trade aggregations as Horizon. Data is available for a given date where trade aggregation information can be provided at the daily, weekly, monthly, or yearly level.
+
+### List Trade Aggregations
+
+```sql
+select
+  day_agg
+  , asset_a_code
+  , asset_a_issuer
+  , trade_count_daily
+  , trade_count_weekly
+  , trade_count_monthly
+  , trade_count_yearly
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar_dbt.trade_agg`
+where true
+  -- there are various columns to filter by
+  and asset_a_code = <asset code>
+  and asset_a_issuer = <asset issuer>
+  and asset_b_code = <asset code>
+  and asset_b_issuer = <asset issuer>
+  and day_agg between <start> and <end>
+```
+
+## Fee Stats
+
+[horizon/fee-stats](/docs/data/horizon/api-reference/aggregations/fee-stats)
+
+Fee stats are used to predict what fee to set for a transaction before submitting it to the network.
+
+### Retrieve Fee Stats
+
+```sql
+select
+  day_agg
+  , fee_charged_max
+  , fee_charged_min
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar_dbt.fee_stats_agg`
+where true
+  and day_agg between <start> and <end>
+```
+
+## getEvents
+
+[rpc/getEvents](/docs/data/rpc/api-reference/methods/getEvents)
+
+Contract events and diagnostic events are used for supplemental information related to a contract invocation
+
+```sql
+select
+  contract_id
+  , topics_decoded
+  , data_decoded
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar.history_contract_events`
+where true
+  -- highly recommended to provide a date range to reduce amount of data queried
+  and closed_at between <start> and <end>
+```
+
+## getLatestLedger
```

Comment:
idk if it's worth us mapping this endpoint to hubble. we're on a delay, so the operator will be retrieving the latest ledger to hubble, not the latest ledger, per se